modules/data-entry/src/main/java/ca/sickkids/ccm/lfs/QueryBuilder.java

```diff
@@ -156,6 +161,23 @@ private String fullTextEscape(String input)
         return input.replaceAll("([\\Q+-&|!(){}[]^\"~*?:\\/\\E])", "\\$1").replaceAll("'", "''");
     }
 
+    /**
+     * Gets the question for a given answer JCR Resource.
+     *
+     * @param res the JCR Resource corresponding to an answer
+     * @return the question string corresponding to the passed answer
+     */
+    private String getQuestion(Resource res) throws ItemNotFoundException, RepositoryException
+    {
+        String questionNodeId = res.getValueMap().get("question", "");
+        if (!"".equals(questionNodeId))
+        {
```

Comment:
Our code style is to put the open curly bracket on a new line only for functions. Make this style change throughout your code.

```suggestion
        if (!"".equals(questionNodeId)) {
```

Reply:
Changed in commit `79ffe93` :heavy_check_mark: